waku/waku_api/rest/relay/handlers.nim

```diff
@@ -135,16 +135,8 @@ proc installRelayApiHandlers*(router: var RestRouter, node: WakuNode, cache: Mes
       if not success:
         return RestApiResponse.internalServerError("Failed to publish: error appending RLN proof to message")
 
-      # validate the message before sending it
-      let result = node.wakuRlnRelay.validateMessageAndUpdateLog(message)
```

Comment:
Why are we removing this `wakuRlnRelay` validation? This is quite important :) Or are we making this validation in another place?

Reply:
Yes!

So now we register the validator here:

https://github.com/waku-org/nwaku/blob/6171f31e896ce3a8aecaaa5f3ad382866e720313/waku/node/waku_node.nim#L990

And this runs in the REST handler here:
https://github.com/waku-org/nwaku/blob/6171f31e896ce3a8aecaaa5f3ad382866e720313/waku/waku_api/rest/relay/handlers.nim#L138

So instead of hardcoding again the RLN validation in the handler, this gets run when going over all the registered validators

Reply:
Beautiful!, I see that the validation is performed in the following place:
https://github.com/waku-org/nwaku/blob/3e65cc18f64c7b8fa0920026e64dbb9507ab0c2e/waku/waku_rln_relay/rln_relay.nim#L331